simp.json.template

```diff
@@ -214,98 +247,141 @@
       "inline" : [
         "echo 'Rebooting...'",
         "if [ -f /usr/sbin/reboot ]; then",
-        "/usr/sbin/reboot",
+          "/usr/sbin/reboot",
         "else",
-        "/sbin/reboot",
+          "/sbin/reboot",
         "fi"
        ]
     },
+
+    // ########################################################################
+    //                  ____  _____ ____   ___   ___ _____
+    //                  |  _ \| ____| __ ) / _ \ / _ \_   _|
+    //                  | |_) |  _| |  _ \| | | | | | || |
+    //                  |  _ <| |___| |_) | |_| | |_| || |
+    //                  |_| \_\_____|____/ \___/ \___/ |_|
+    //
+    // ########################################################################
+
+    // Upload puppet code
     {
-## Pause a little to let the system boot up all the way then
-## Copy setup manifests
+      // Pause a little to let the system boot up all the way
       "pause_before": "300s",
       "type": "file",
-      "source": "./manifests",
+      "source": "./puppet",
       "destination": "/var/local/simp"
     },
+
+    // Run puppet again to get closer to a clean run (take 1)
     {
-## Run puppet again to get closer to a clean run (take 1)
       "type": "puppet-server",
       "ignore_exit_codes": true,
       "extra_arguments": "--test",
       "puppet_bin_dir": "/opt/puppetlabs/bin"
     },
+
+    // Run puppet again to get closer to a clean run (take 2)
     {
-## Run puppet again to get closer to a clean run (take 2)
       "type": "puppet-server",
       "ignore_exit_codes": true,
       "extra_arguments": "--test",
       "puppet_bin_dir": "/opt/puppetlabs/bin"
     },
-## TODO:  I had to wrap the ruby scripts in shell scripts because it
-## runs them as shells.  Need to figure out the correct
-## execute_command for executing a ruby script
+
+    // FIXME: REMOVE THIS LATER IT IS ONLY FOR DEBUGGING
+    {
+      "type": "shell",
+      "remote_folder": "/var/local/simp/scripts",
+      "inline": [ "sudo yum install -y tree; :", "tree /var/local/simp" ]
+    },
+
+    // Grant all uploaded scripts premission to execute on the server
+    {
+      "type": "shell",
+      "remote_folder": "/var/local/simp/scripts",
+      "inline": [ "sudo find /var/local/simp/scripts -type f -name '*.sh' -exec chmod +x {} \\;" ]
+    },
+
+    // TEST: Check settings after bootstrap.
+    //
+    // TODO:  I had to wrap the ruby scripts in shell scripts because it
```

Comment:
looks like you solved this problem and can remove the comment

Reply:
@lnemsick-simp Not yet—`scripts/tests/check_settings.sh` still calls `/scripts/tests/check_settings.rb`